packages/babel-plugin-external-helpers/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "babel-plugin-external-helpers",
   "version": "6.8.0",
-  "description": "",
+  "description": "Babel has a few helper functions that’ll be placed at the top of the generated code if needed so it’s not inlined multiple times throughout that file",
```

Comment:
I think this is already too much info. I think it is enough to say `("This plugin" see other comment )Contains helper functions that’ll be placed at the top of the generated code.`